### PR TITLE
feat(DMVP-6117): Add/Update widgets for service block

### PR DESCRIPTION
**Pull request by greta-dasmeta-com**
## Summary
The service block is modified based on our currently used dashboards

### CODE REVIEW

File: modules/dashboard/modules/widgets/pod/memory/base.tf [hunk 1]
```diff
@@ -18,9 +19,9 @@ module "base" {
   }
 
   metrics = [
-    { label = "AVG", expression = "avg(container_memory_usage_bytes{container=\"${var.container}\", namespace=\"${var.namespace}\"}) by (container) / 1024 / 1024" },
-    { label = "Request", expression = "avg(kube_pod_container_resource_requests{container=\"${var.container}\", namespace=\"${var.namespace}\", resource=\"memory\"}) / 1024 / 1024" },
-    { label = "Limit", expression = "avg(kube_pod_container_resource_limits{container=\"${var.container}\", namespace=\"${var.namespace}\", resource=\"memory\"}) / 1024 / 1024" },
-    { label = "MAX", expression = "max(container_memory_usage_bytes{container=\"${var.container}\", namespace=\"${var.namespace}\"}) by (container) / 1024 / 1024" },
+    { label = "Avg", color = "FFC300", expression = "avg(rate(container_memory_usage_bytes{pod=~\"${var.pod}-[^-]+-[^-]+$\", namespace=\"${var.namespace}\"}[${var.period}m])) / 1024 / 1024 * 100" },
```
Comment (viktoryathegreat) [inline modules/dashboard/modules/widgets/pod/memory/base.tf:22]: Please, review the calculation of memory and its unit. This query shows KBs on the widget, but we don't use KBs there.

File: modules/dashboard/widgets-deployment.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  # deployment
+  deployment       = each.value.deployment
+  cluster   = try(each.value.cluster, null)
+  namespace = each.value.namespace
+
+  account_id        = each.value.account_id
+  region            = each.value.region
+  anomaly_detection = each.value.anomaly_detection
+  anomaly_deviation = each.value.anomaly_deviation
+}
```
Comment (viktoryathegreat) [inline modules/dashboard/widgets-deployment.tf:20]: Please, add a new line here.

File: modules/dashboard/widgets-pod.tf [hunk 3]
```diff
@@ -19,3 +19,43 @@ module "pod_cpu_widget" {
...
+  # pod
+  pod       = each.value.pod
+  cluster   = try(each.value.cluster, null)
+  namespace = each.value.namespace
+
+  account_id        = each.value.account_id
+  region            = each.value.region
+  anomaly_detection = each.value.anomaly_detection
+  anomaly_deviation = each.value.anomaly_deviation
+}
```
Comment (viktoryathegreat) [inline modules/dashboard/widgets-pod.tf:61]: New line, again.

File: modules/dashboard/modules/widgets/container/network/base.tf [hunk 4]
```diff
@@ -18,10 +19,8 @@ module "base" {
   }
 
 
-  metrics = concat([
-    { label = "In(container)", expression = "sum(rate(container_network_receive_bytes_total{namespace=\"${var.namespace}\", pod=~\"${var.container}-.*\"} [${var.period}m])) / 100000" },
-    { label = "Out(container)", expression = "sum(rate(container_network_transmit_bytes_total{namespace=\"${var.namespace}\", pod=~\"${var.container}-.*\"} [${var.period}m])) / 100000" },
-    ],
-    var.host != null ? [{ label = "In(ingress)", expression = "sum(rate(nginx_ingress_controller_bytes_sent_bucket{host=\"${var.host}\"}[${var.period}m]))" }] : [],
-  )
+  metrics = [
```
Comment (viktoryathegreat) [inline modules/dashboard/modules/widgets/container/network/base.tf:22]: Please, bring back the old queries, those were specified for some purpose. Also, make the labels in small letters(Received, Sent).

**Review by mrdntgrn — DISMISSED** [review]

**Review by mrdntgrn — APPROVED** [review]

**Review by mrdntgrn — APPROVED** [review]

**Pull request merged**